### PR TITLE
Add backup location to storage backup schedule.

### PR DESCRIPTION
**Pull request by nvthongswansea**
Ref #210. Changes:
- Add storage backup location API endpoint.
- Add method `GetStorageBackupLocationList` to fetch a list of available location used to store backups.
- Allow users to set location for backup in backup schedule.
- Update unit test.

### CODE REVIEW

File: storage_backup_schedule.go [hunk 1]
```diff
@@ -62,6 +62,12 @@ type StorageBackupScheduleProperties struct {
 
 	// Status of the schedule.
 	Active bool `json:"active"`
+
+	// The Location where your backup is stored.
```
Comment (itakouna) [inline storage_backup_schedule.go:66]: The UUID of the location where the backup is stored.

File: storage_backup_schedule.go [hunk 2]
```diff
@@ -100,6 +106,9 @@ type StorageBackupScheduleCreateRequest struct {
...
```
Comment (itakouna) [inline storage_backup_schedule.go:110]: same here

File: storage_backup_schedule.go [hunk 3]
```diff
@@ -132,6 +141,25 @@ type StorageBackupScheduleUpdateRequest struct {
 	Active *bool `json:"active,omitempty"`
 }
 
+// StorageBackupLocationList holds a list of available location to store your backup.
```
Comment (itakouna) [inline storage_backup_schedule.go:144]: contains

File: storage_backup_schedule.go [hunk 4]
```diff
@@ -217,3 +245,23 @@ func (c *Client) DeleteStorageBackupSchedule(ctx context.Context, storageID, sch
 	}
 	return r.execute(ctx, *c, nil)
 }
+
+// GetStorageBackupLocationList gets a list of available locations to store your backup.
```
Comment (itakouna) [inline storage_backup_schedule.go:249]: returns

**Comment by nvthongswansea** [comment]
@itakouna Done rewording.

File: storage_backup_schedule.go [hunk 5]
```diff
@@ -63,14 +63,14 @@ type StorageBackupScheduleProperties struct {
 	// Status of the schedule.
 	Active bool `json:"active"`
 
-	// The Location where your backup is stored.
+	// The UUD of the location where your backup is stored.
```
Comment (itakouna) [inline storage_backup_schedule.go:66]: UUID ;)

**Pull request merged**